packages/@aws-cdk/cx-api/lib/context/availability-zones.ts

```diff
@@ -0,0 +1,22 @@
+export const AVAILABILITY_ZONE_PROVIDER = 'availability-zones';
+
+/**
+ * Query to hosted zone context provider
+ */
+export interface AvailabilityZonesContextQuery {
+  /**
+   * Query account
+   */
+  account?: string;
+
+  /**
+   * Query region
+   */
+  region?: string;
+
```

Comment:
Redundant

Reply:
It's not, though I suppose I could model it differently.